pkg/checks/traceroute/config.go

```diff
@@ -0,0 +1,39 @@
+package traceroute
+
+import (
+	"fmt"
+	"net"
+	"net/url"
+	"time"
+
+	"github.com/caas-team/sparrow/pkg/checks"
+)
+
+type Config struct {
```

Comment:
Please comment the struct and the fields, as we want to create automatic documentation for the structs down the line ;)